pwndbg/lib/arch.py

```diff
@@ -2,8 +2,49 @@
 
 import struct
 import sys
+import typing
 from typing import Literal
 
+# Names of arches that Pwndbg supports
+# The names that GDB/LLDB recognize are converted to a name in this list
+PWNDBG_SUPPORTED_ARCHITECTURES_TYPE = Literal[
+    "x86-64",
+    "i386",
+    "i8086",
+    "mips",
+    "aarch64",
+    "arm",
+    "armcm",
+    "iwmmxt",
+    "rv32",
+    "rv64",
+    "sparc",
+    "powerpc",
+    "loongarch64",
+]
+
+PWNDBG_SUPPORTED_ARCHITECTURES: list[PWNDBG_SUPPORTED_ARCHITECTURES_TYPE] = list(
+    typing.get_args(PWNDBG_SUPPORTED_ARCHITECTURES_TYPE)
+)
+
+# mapping between pwndbg and pwntools arch names
+PWNLIB_ARCH_MAPPINGS = {
+    "x86-64": "amd64",
+    "i386": "i386",
+    "i8086": "none",
+    "mips": "mips",
+    "aarch64": "aarch64",
+    "arm": "arm",
+    "iwmmxt": "arm",
+    "armcm": "thumb",
```

Comment:
```suggestion
    "arm": "arm",
    "armcm": "thumb",
```
removed `iwmmxt` (probably it is already handled by `arm` name)